Models/Feed/CheckInEvent.m

```diff
@@ -10,16 +10,15 @@
 
 @implementation CheckInEvent
 - (instancetype)initWithParseObject:(PFObject *)object {
```

Comment:
if you're not doing any additional initialization here, I don't think you even need to override genTabContentListViewSurface